content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+
+1. Typing in the search bar above the table
+2. Setting a default filter in the widget's configuration This filter will be
+   persisted and used when viewing the widget again.
+3. Applying a local filter
+
+   By clicking the "Filter" button in the top-right
+   corner of the widget, one can select a local filter. It will override the
+   default filter picked in the widget's configuration. The local filter will be
+   reset when viewing the widget again.
+
+### Deployments Map
+
+Clicking the _Map_ button in the top part of the widget will open an additional
+map section in the top part of the widget. The map presents deployments in the
+sites that are attached to them.
+
+<!-- TODO: screenshot map -->
+
+The height of the map section, as well as whether it should be open by default
+can be changed using the widget's configuration.
+
+The color of the marker representing a deployment depends on the deployment's
+state:
+
+1. Blue - _good_
+2. Yellow - _in progress_
+3. Red - _requires attention_
+
+The map supports zooming (when scrolling using the mouse or when using the zoom
+buttons in the top-left part of the map) and panning (when dragging using the
+mouse).
+
+Deployments that are close together are grouped into a cluster that displays the
+number of deployments in that clustered group. The color of the cluster icon
+depends on the worst state of deployments inside of it. Clicking the marker will
+expand the cluster (either by zooming in or by breaking the cluster apart into
+individual markers).
+
+The currently selected deployment is marked by an additonal circle underneath
+the marker. Such a deployment is not clustered, meaning it will appear outside
+of any cluster, even if it is close to other deployments.
+
+Clicking a deployment marker will select it in the table and affect the contents
+of the [Details pane]({{< relref "#details-pane" >}}).
+
+Hovering over a deployment marker will show a tooltip with additonal
+information:
+
+- deployment name
+- deployment blueprint name
+- site name
+- deployment status
+- the counts and statuses of child deployments
+
+<!-- TODO: screenshot tooltip -->
+
+### Bulk Actions
+
+The Deployments View widget also has a _Bulk Actions_ button in the top-right
+corner of the widget. It allows invoking actions over all deployments that match
+the current filter criteria.
+
+{{% note %}}
+Keep in mind the actions will be invoked also for deployments that are on other
+pages of the table (not only the deployments that are visible in the current
+table page).
+{{% /note %}}
+
+#### Deploy On
+
+The _Deploy On_ bulk action will show a modal that guides the user through
+deploying additional services on the environments that are matched by the
+current filter.
+
+After selecting the base blueprint for the new deployments and providing any
+additional labels, there will be a new child deployment created for each
+deployment matched by the current filter.
```

Comment:
each generated deployment will be labeled with the Environment ID of the environment on which it is deployed as a parent label, and the capabilities of that parent environment can be retrieved using the [get-environment-capability]({{< relref "developer/blueprints/spec-intrinsic-functions/#get-environment-capability" >}}) intrinsic function.